kornia/color/rgb.py

```diff
@@ -120,11 +120,12 @@ def bgr_to_rgba(image: torch.Tensor, alpha_val: Union[float, torch.Tensor]) -> t
     return rgb_to_rgba(x_rgb, alpha_val)
 
 
-def rgba_to_rgb(image: torch.Tensor) -> torch.Tensor:
+def rgba_to_rgb(image: torch.Tensor, bg_rgb: Tuple = (0, 0, 0)) -> torch.Tensor:
```

Comment:
`bg_rgb` reads a bit difficult. Maybe we should just name it to `background`

Reply:
Also should specify that values are expected to be in range [0,1] (or handle in code accordingly if 0,255)

Reply:
Any naming suggestions from ```PIL``` or ```opencv```? Or naming  it as full as ```background_color```.

The whole lib shall support floating point range only for the differentiability. Probably adding a wrapper like:
https://github.com/kornia/kornia/blob/71acf455ee36f2050b7be5ea993b6db773f502eb/kornia/augmentation/utils/helpers.py#L10-L28

Reply:
PIL or opencv doesn't directly support specifying a background color during conversion (have to do it manually with alpha blending prior or img copy/paste etc). 
[scikit-image](https://github.com/scikit-image/scikit-image/blob/2bbfce4786102a3a9e85839a52303e858346a08e/skimage/color/colorconv.py#L131-L187) does this and uses `background` name.

Reply:
`background` suits better as in `scikit-image` implementation. @shijianjian he wrapper thing should go in a separated PR